ui/Modal/Funding/Pool/Withdraw.svelte

```diff
@@ -14,17 +14,18 @@
   } from "../../../src/funding/pool";
   import { ValidationStatus } from "../../../src/validation";
 
-  import { BigNumber } from "ethers";
+  import Big from "big.js";
 
   if ($store === null) pop();
 
   // Validate the amount beign withdrawn
   let validatingAmount = false;
   let amount = "";
-  let validation = balanceValidationStore(0);
+  let balance = Big(0);
+  let validation = balanceValidationStore(balance);
 
   $: {
-    const balance = $store?.data.unwrap()?.balance || 0;
+    balance = $store?.data.unwrap()?.balance || balance;
```

Comment:
Wouldn't it make more sense to do this?
```suggestion
const balance = $store?.data.unwrap()?.balance || Big(0);
```
And remove the `  let balance = Big(0);` above, since it's not used anywhere else?

Reply:
I like that but then the editor will go :banana: saying that `balance` is not declared. Thanks anyway!